canonical/snapcraft.io
# fix: Update homepage search to use the new store page results

## Done
- Updated the homepage search to use the store page for results
- Removed the old search view

## How to QA
- Go to https://snapcraft-io-4631.demos.haus/
- Search for a snap
- Check that you are taken to https://snapcraft-io-4631.demos.haus/store?q=QUERY
- Go to https://snapcraft-io-4631.demos.haus/safasdf
- Search for a snap
- Check that the form on the 404 page takes you to https://snapcraft-io-4631.demos.haus/store?q=QUERY
- Go to https://snapcraft-io-4631.demos.haus/search?q=code
- Check that you are redirected to https://snapcraft-io-4631.demos.haus/store?q=code

## Testing
- [x] No testing required (explain why): Removing an old view

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-11333
Fixes #4623
